spec/system/hiring_staff_can_publish_a_vacancy_as_a_school_group_spec.rb

```diff
@@ -135,15 +138,113 @@
         fill_in_job_location_form_field(vacancy)
         click_on I18n.t('buttons.continue')
 
-        fill_in_school_form_field(school)
+        fill_in_school_form_field(school_1)
         click_on I18n.t('buttons.continue')
 
         fill_in_job_specification_form_fields(vacancy)
         click_on I18n.t('buttons.continue')
 
         expect(Vacancy.last.state).to eql('create')
-        expect(Vacancy.last.readable_job_location).to eql(school.name)
-        expect(Vacancy.last.organisation).to eql(school)
+        expect(Vacancy.last.readable_job_location).to eql(school_1.name)
+        expect(Vacancy.last.organisation).to eql(school_1)
+        activity = Vacancy.last.activities.last
+        expect(activity.session_id).to eql(session_id)
+        expect(activity.key).to eql('vacancy.create')
+        expect(activity.parameters.symbolize_keys).to include(job_title: [nil, vacancy.job_title])
+      end
+    end
+  end
+
+  context 'when job is located at multiple schools in the school group' do
+    let(:vacancy) do
+      build(:vacancy, :at_multiple_schools, :complete)
+    end
+
+    describe '#job_location' do
+      context 'when only 1 school is selected' do
+        scenario 'displays error message and vacancy is not created' do
+          visit new_organisation_job_path
+
+          expect(page).to have_content(I18n.t('jobs.current_step', step: 1, total: 8))
+          within('h2.govuk-heading-l') do
+            expect(page).to have_content(I18n.t('jobs.job_location'))
+          end
+
+          fill_in_job_location_form_field(vacancy)
+          click_on I18n.t('buttons.continue')
+
+          expect(page).to have_content(I18n.t('jobs.current_step', step: 1, total: 8))
+          within('h2.govuk-heading-l') do
+            expect(page).to have_content(I18n.t('jobs.job_location'))
+          end
+
+          expect(Vacancy.count).to eql(0)
```

Comment:
There is also `not_to change` (equivalent to `to_not change`) - maybe a slightly more reliable way of putting it in general, because we are testing the difference in the count, not the absolute value of the count.

Reply:
I don't mind which you choose to use. There are many `expect(Vacancy.count).to eql(0)`, some of which are written by you, some of which are old, so whatever you choose let's regularize

Reply:
It would require you to pass in the interaction as a code block:

```ruby
expect { do_something }.to change { object.attribute }
```